docs/user/operator-hub.adoc

```diff
@@ -0,0 +1,271 @@
+:source-highlighter: pygments
+
+# odo integration with Operator Hub
+---
+
+When working in experimental mode, odo provides the ability to work with
+link:https://www.openshift.com/learn/topics/operators[Operators] installed on
+the cluster. It allows listing of Operators, creation of services from CRD
+(Custom Resource Definitions) provided by the Operators, printing the YAML
+definition and providing custom YAML definition to start the service from a
+CRD.
+
+[NOTE]
+====
+Installation of Operators is not a part of odo workflow. It is something that
+your OpenShift/Kubernetes cluster administrator should be able to do for you. 
+====
+
+=== Prerequisites
+
+- `odo` is installed.
+- Required Operators are installed in the project/namespace by cluster
+  administrator.
+- Experimental mode is enabled. For every command (other than those for
+  <<create-project,project creation>>) mentioned in this document to work, we
+  first need to enable the experimental mode. This can be done in
+  different ways. Make sure that you have installed latest version of odo and
+  perform any one of the following actions.
+
+* Enable experimental mode in odo preferences:
++
+[source,shell]
+----
+$ odo prefrence set Experimental true
+----
+
+
+* Export the environment variable `ODO_EXPERIMENTAL`:
++
+[source,shell]
+----
+$ export ODO_EXPERIMENTAL=true
+----
+
+* Prefix every command mentioned below with `ODO_EXPERIMENTAL=true`. For
+  example, to list the Operators installed in current project:
++
+[source,shell]
+----
+$ ODO_EXPERIMENTAL=true odo catalog list services
+----
+
+=== [[create-project]]Creating a project
+Create a project to keep your source code, tests, and libraries organized in a
+separate single unit.
+
+1. Log in to the Kubernetes/OpenShift cluster:
++
+[source,shell]
+----
+$ odo login -u developer -p developer
+----
+
+2. Create a project:
++
+[source,shell]
+----
+$ odo project create myproject
+ ✓  Project 'myproject' is ready for use
+ ✓  New project created and now using project : myproject
+----
+
+=== [[list-operators]]Listing the Operators
+
+To list the Operators installed in current project, execute below command:
+
+[source,shell]
+----
+$ odo catalog list services
+----
+
+It will list the
+link:https://docs.openshift.com/container-platform/4.3/operators/olm-what-operators-are.html[Operators]
+and the services, or
+link:https://docs.openshift.com/container-platform/4.3/operators/crds/crd-extending-api-with-crds.html#crd-custom-resource-definitions_crd-extending-api-with-crds[CRD
+(Custom Resource Definitions)], provided by these Operators. For example, we
+have installed etcd and MongoDB Operators and the output we get is like below:
+
+[source,shell]
+----
+$ odo catalog list services                       
+Operators available in the cluster
+NAME                          CRDs
+etcdoperator.v0.9.4           EtcdCluster, EtcdBackup, EtcdRestore
+mongodb-enterprise.v1.4.5     MongoDB, MongoDBUser, MongoDBOpsManager
+----
```

Comment:
No need to have this section. Instead, add the output of this into line 75:

https://github.com/openshift/odo/pull/3029/files#diff-15071c5bfeaad296b2a7b29032ce0d56R75

And then go into detail after with the following paragraphs / sentences.